content/en/tracing/troubleshooting/diagnostic_tool.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using the .NET diagnostic tool for troubleshooting
+kind: documentation
+---
+
+If after installing the .NET tracer your application does not produce traces as expected, you can try running the diagnostic tool for basic troubleshooting. 
+
+The tool can be installed in two different ways:
+
+- Using the .NET SDK by running the command:
+```
+dotnet tool install -g dd-trace
+```
+- Or by direct download by picking the right version [from the release page](https://github.com/DataDog/dd-trace-dotnet/releases)
+**TODO: We probably want to setup a short url?**
+
+
+## Process diagnostics 
+
+For most applications, you should use the process diagnostics. For that, you need to make sure the application is running, and get the pid. You can then give it to the dd-trace tool:
+```
+dd-trace check process <pid>
```

Comment:
Is it possible to have an example output section for the commands? It would help to know what kind of information is emitted. The actual values can be obfuscated if they contain any sensitive info.